examples/Jaeger.Example.WebApi/Jaeger.Example.WebApi.csproj

```diff
@@ -7,6 +7,12 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
     <PackageReference Include="OpenTracing.Contrib.NetCore" Version="0.4.0" />
+
+    <!-- NOTE:
+        In case you're building this application standalone, use below PackageReference 
+        with appropriate an appropriate version instead of relative-path ProjectReference 
```

Comment:
"with [-appropriate] an appropriate version " There is a second "appropriate" that needs to be removed. Other than that, I'm fine with the suggestion.

A more detailed tutorial can be found at https://github.com/yurishkuro/opentracing-tutorial/tree/master/csharp Not sure if it is linked somewhere, if not, we also may should add a link in the README.md to it.

Reply:
@Falco20019 I've fixed it now. At first sight, I couldn't see if there is any link to tutorial repo.